tdrs-backend/tdpservice/settings/common.py

```diff
@@ -294,7 +294,7 @@ class Common(Configuration):
     ###
 
     # Flag for local testing to enable AV Scans
-    CLAMAV_NEEDED = get_required_env_var_setting('CLAMAV_NEEDED')
+    CLAMAV_NEEDED = os.getenv('CLAMAV_NEEDED', True)
```

Comment:
This wasn't working in the false case. It seems like it is seeing a string of length greater than 0 and accepting that as true. I found this solution:
```suggestion
    CLAMAV_NEEDED = bool(os.getenv('CLAMAV_NEEDED', 'True').lower() in ('true', '1', 't'))
```

Source: https://stackoverflow.com/questions/63116419/evaluate-boolean-environment-variable-in-python